README.md

```diff
@@ -1,7 +1,32 @@
-# helptheducks.com
+# DxE Petitions Sites monorepo
```

Comment:
nit: not sure if we want to distinguish before "normal" "petition" sites, like the petition on righttorescue.com and this new type that actually sends emails to a target.